### ISSUE TITLE
add optional sha265 sum support to download worker

### ISSUE THREAD

**Issue by derpeter** [voc/voctopublish#63]

**Comment by a-tze** [comment]
maybe also just fire ffprobe after download?

**Comment by derpeter** [comment]
can i add the ouput to the setTicketDone call? https://github.com/voc/voctopublish/blob/htmlupload/src/api_client/c3tt_rpc_client.py#L183

**Comment by a-tze** [comment]
Yes, it takes a string as optional second argument which is added as log text:

https://github.com/crs-tools/crs-scripts/blob/master/scripts/script-F-postprocessing-upload.pl#L62

**Comment by derpeter** [comment]
perfekt, will do it this way.
